File: php/bubble_sort.php
<?php
$array = [2,85,1,95,76,2,9];

print_r(array_values($array));

for($i=0;$i<sizeof($array);$i++){

    for($j=0;$j<sizeof($array);$j++){
        if($array[$j]>$array[$i]){
            $temp = $array[$i];
            $array[$i] = $array[$j];
            $array[$j] = $temp;
        }
    }
}

print_r($array);

?>

File: python/bubble_sort.py

list = [86, 7, 5, 6, 8, 5, 1, 25, 3]
for i in range(len(list)):
    for j in range(len(list)):
        if list[j]>list[i]:
            temp = list[i]
            list[i] = list[j]
            list[j] = temp
print(list,end=" ")